CovertInfiltration/Src/CovertInfiltration/Classes/UISS_InfiltrationItem.uc

```diff
@@ -0,0 +1,61 @@
+//---------------------------------------------------------------------------------------
+//  AUTHOR:  (Integrated from Long War 2)
+//           Adapted for overhaul by ArcaneData
+//  PURPOSE: Item linking to X2ObjectiveListItem for displaying shadowed text
+//---------------------------------------------------------------------------------------
+//  WOTCStrategyOverhaul Team
+//---------------------------------------------------------------------------------------
+
+class UISS_InfiltrationItem extends UIPanel;
+
+var UIPanel TitleIcon;
+
+simulated function UISS_InfiltrationItem InitObjectiveListItem(optional int InitX = 0, optional int InitY = 0)
+{
+	InitPanel(); 
```

Comment:
Please take an `optional name InitName` argument and pass it to `InitPanel(InitName)`